training/codelabs_k2_day4/codelabs_day4_Tra/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>NguyenThiThuTra</title>
+    <style>
+        li {
+            display:inline;
+        }
+    </style>
+    <style>
+       body {
+           background-color: black;
+
+       } 
+    </style>
+</head>
+<body>
+    
+    <ul>
+        <a href="https://www.facebook.com/"><img src="png/Capture11..PNG"></a>
+        <input type="text" placeholder="Tim kiem thong tin" >
+        <li><a href="https://www.facebook.com/"><img src="png/Capture1.PNG" alt="Facebook"></a></li>
+        <li><a href="https://www.facebook.com/watch"><img src="png/Capture2.PNG" alt="Facebook"></a></li>
+        <li><a href="https://www.facebook.com/marketplace?ref=app_tab"><img src="png/Capture3.PNG" alt="Facebook"></a></li>
+        <li><a href="https://www.facebook.com/groups/feed/"><img src="png/Capture4.PNG" alt="Facebook"></a></li>
+        <li><a href="https://www.facebook.com/gaming/?ref=games_tab"><img src="png/Capture5.PNG" alt="Facebook"></a></li>
+        <li><a href="https://www.facebook.com/"><img src="png/Capture6.PNG" alt="Facebook"></a></li>
+        <li><a href="https://www.facebook.com/"><img src="png/Capture9.PNG" alt="Facebook"></a></li>
+        <li><a href="https://www.facebook.com/"><img src="png/Capture7.PNG" alt="Facebook"></a></li>
+        <li><a href="https://www.facebook.com/"><img src="png/Capture8.PNG" alt="Facebook"></a></li>
+        <li><a href="https://www.facebook.com/"><img src="png/Capture10.PNG" alt="Facebook"></a></li>
```

Comment:
Toàn bộ các ảnh png này chưa phải là png icon, ảnh icon chỉ chứa icon không được chứa các vùng đen hoặc gì khác ngoài icon. Toàn bộ những thứ này em phải style nếu em để như vậy khi đổi background sang màu khác sẽ không được , em xem lại phần này nhé, tìm hiểu padding và margin trong css và thay đổi lại các icon này

Reply:
vâng ạ, em sẽ làm lại ạ